### DISCUSSION TITLE
I Am Publishing H.265 but Not Receiving an M3U8 Stream

### DISCUSSION THREAD

**Discussion by Maheshbhadiya** [bluenviron/mediamtx#2967] [Questions]
### Question

Hello everyone,

I am streaming content using the following command:

ffmpeg -rtsp_transport tcp -i rtsp://admin:1234@192.168.2.222:9008/profile1 -pix_fmt yuv420p -c:v libx265 -preset ultrafast -tune zerolatency -f rtsp rtsp://3.143.143.143:8554/live/profile1

However, when I attempt to access http://3.143.143.143:8554/live/profile1/index.m3u8, it only returns a blank screen.

Please guide me if I have made any mistakes.

**Comment by aler9** [comment]
Hello, the command you posted seems right to me, however it's impossible to tell what is going on without a server log, a FFmpeg log and a log from the browser. If you still have problems, open another question or bug report and attach both.

**Reply by Maheshbhadiya** [reply]
Hi, thank you for your response. I've discovered that while H.265 is supported with low latency, it is not compatible with MPEG-TS. Additionally, when using low latency settings, the latency increases significantly to around 30-45 seconds.
Could you please help me to improve latency with lowLatency?

[a comment by an automated account is not shown]
